src/main/scala/uk/gov/ons/bi/dataload/exports/HmrcBiCsvExtractor.scala

```diff
@@ -3,7 +3,7 @@ package uk.gov.ons.bi.dataload.exports
 import org.apache.log4j.Level
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.{Column, DataFrame}
-import org.apache.spark.sql.functions.{concat, concat_ws, explode, lit}
+import org.apache.spark.sql.functions.{concat, concat_ws, explode, lit, col}
```

Comment:
Was the col function needed in the import, no evidence that you're using the import